lib/epi_deploy/release.rb

```diff
@@ -23,7 +23,7 @@ def create!
 
         self.tag = "#{date_and_time_for_tag}-#{git_wrapper.short_commit_hash}-v#{new_version}"
         git_wrapper.tag self.tag
-        git_wrapper.push 'master'
+        git_wrapper.push git_wrapper.current_branch
```

Comment:
I like this a lot, nice change which is flexible for allowing multiple branch names in the future.

Reply:
Thanks :)